tests/test_preprocessors.py

```diff
@@ -71,4 +71,16 @@ def test_insert_action(preprocessor):
         preprocessor.insert_action("new4", action)
 
 
+def test_trace_off(preprocessor, row):
+    x, sample_info = preprocessor.forward(row)
+    assert sample_info["_trace"] is None
+
+
+def test_trace_on(preprocessor, row):
+    x, sample_info = preprocessor.forward(row, trace=True)
+    assert (
+        sample_info["_trace"].index == sample_info["_preprocessor"].pipeline.index
+    ).all()
+
+
```

Comment:
please add a test for one pipeline element to ensure the type of sample_info["_trace"][i] is the expected data type (for instance, Audio after the load_audio action)